ztp/gitops-subscriptions/argocd/example/policygentemplates/group-du-3node-ranGen.yaml

```diff
@@ -43,20 +48,3 @@ spec:
           pages:
             - size: 1G
               count: 32
-    - fileName: TunedPerformancePatch.yaml
-      policyName: "config-policy"
-      spec:
-        profile:
-          - name: performance-patch
-            # The 'include' line must match the PerformanceProfile metadata.name above (openshift-node-performance-${metadata.name})
-            # And the cmdline_crash CPU set must match the 'isolated' set in the PerformanceProfile above
-            data: |
-              [main]
-              summary=Configuration changes profile inherited from performance created tuned
-              include=openshift-node-performance-openshift-node-performance-profile
-              [bootloader]
-              cmdline_crash=nohz_full=2-19,22-39
-              [sysctl]
-              kernel.timer_migration=1
-              [service]
-              service.stalld=start,enable
```

Comment:
How is the start/enable of stalld accounted for?

Reply:
Since we are removing `nohz_full` from `tuned`, there's no need account for the cpu starvation via `stalld`. cc @browsell